README.md

```diff
@@ -161,6 +161,11 @@ General configuration is grouped in a `[bumpversion]` section.
   Also available as command-line flag `tag-name`.  Example usage:  
   `bump2version --tag-name 'release-{new_version}' patch`
 
+  In addition, it is also possible to provide a tag message by using `--tag-message TAG_MESSAGE`. Example usage:
+  `bump2version --tag-name 'release-{new_version}' --tag-message "Release {new_version}" patch`
+
+  * If neither tag message or sign tag is provided, `bumpversion` uses a `lightweight` tag. Otherwise, it utilizes an `annotated` tag.
```

Comment:
Can you add: `see also: https://git-scm.com/book/en/v2/Git-Basics-Tagging`

Reply:
Is this also true for Mercurial?

Reply:
The link has been added, I am not sure about Mercurial, I don't think so.